app/models/patient.rb

```diff
@@ -1018,7 +1017,7 @@ def send_assessment
 
     # Check last_assessment_reminder_sent before enqueueing to cover potential race condition of multiple reports
     # being sent out for the same monitoree.
-    return unless last_assessment_reminder_sent_eligible? || send_now
+    return unless last_assessment_reminder_sent_eligible?
```

Comment:
send_now was removed from this method a long time ago

Reply:
Good catch. Can you also please update the method comment here which also still references `send_now`?